package.json

```diff
@@ -11,7 +11,9 @@
     "startb": "conda activate dlplayground && python -m backend.driver",
     "secrets": "conda activate dlplayground && python -m backend.aws_helpers.aws_secrets_utils.build_env",
     "secrets:deploy": "python -m backend.aws_helpers.aws_secrets_utils.build_env",
-    "docker:build": "docker build -t dlp ."
+    "docker:build": "docker build -t dlp .",
+    "dev:variable": "cd frontend/playground-frontend && dotenv set MODE dev",
+    "prod:variable": "cd frontend/playground-frontend && dotenv set MODE prod"
```

Comment:
This is good but I recommend we have a default mode. Right now, most devs are starting using `npm run startf`, which isn't specific whether it's prod or dev. I recommend changing `startf` in line 10 to `"start": "cd frontend/playground-frontend && npm run dev:variable"`